integration/cli-app/files/Samples.sol

```diff
@@ -0,0 +1,19 @@
+pragma solidity ^0.4.24;
+
+contract Foo {
+  function say() pure returns(string) {
+    return "Foo";
+  }
+}
+
+contract Bar {
+  function say() pure returns(string) {
+    return "Bar";
+  }
+}
+
+contract Baz {
+  function say() pure returns(string) {
+    return "Baz";
+  }
+}
```

Comment:
What about renaming to `integration` to `test`? IMO it will be clearer about its content, do you agree?